.github/ISSUE_TEMPLATE/request.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 💡 Feature Request
+about: I have a suggestion (and might want to implement myself 🙂)!
+---
+
+<!--
+
+Vote on feature requests by adding a 👍. This helps maintainers prioritize what
+to work on.
+
+* Please fill out this template with all the relevant information so we can
+  understand what's going on and fix the issue. We appreciate bugs filed and PRs
+  submitted!
+
+-->
+
+### Describe the feature you'd like
+
+<!--
+A clear and concise description of what you want to happen. Add any considered
+drawbacks.
+-->
+
+### Suggested implementation
```

Comment:
Similar issue with the Bug report and the implementation. In addition, you will end up writing the solution in the ticket and discuss solutions in issues instead of PRs. I'd prefer to do it there TBH